main.py

```diff
@@ -19,12 +19,14 @@ def student():
 
 @app.route('/result',methods = ['POST', 'GET'])
 def result():
+   global channel_name
```

Comment:
use cred.py as future migration to db will be easier

Reply:
done